q-verse/lms/templates/courseware/program_marketing.html

```diff
@@ -97,7 +97,7 @@ <h2>${program['subtitle']}</h2>
                       oldPrice=full_program_price_format.format(program['discount_data']['total_incl_tax_excl_discounts'])
                     )}</span
                   ><span aria-label="${_('Discounted Price')}" class="discount">
-                      ${currency_symbol}${newPrice}
+                      ${currency_symbol}${full_program_price}
```

Comment:
Why are we replacing the `newPrice` with `full_program_price`?

Reply:
At first "newPrice" was the variable used in formating code as given below:

```
<span aria-label="${_('Discounted Price')}" class="discount">
                      ${Text(_('${newPrice}')).format(
                        newPrice=full_program_price,
                      )}
                  </span>
```

After removing the above formatting code, "newPrice" variable left mistakenly.